src/Controller/CRUDController.php

```diff
@@ -983,6 +985,8 @@ final protected function renderWithExtraParams(string $view, array $parameters =
      * @param array<string, mixed> $parameters
      *
      * @return array<string, mixed>
+     *
+     * @deprecated since sonata-project/admin-bundle version 4.x
```

Comment:
I was thinking about adding
```
NEXT_MAJOR: Remove the code inside this function
```
But keeping it.

It's maybe a better idea to deprecate this method, but then we need at least to add a message in the Upgrade Note. And a `NEXT_MAJOR` comment on this one.

@jordisala1991 Do you know if we override this method in other sonata bundles ?